Fix Segmentation fault on moving hashtable internal pointer of zend_empty_array
--- a/blitz-0.10.6/blitz.c
+++ b/blitz-0.10.6/blitz.c
@@ -5058,6 +5058,7 @@
     /* set works differently for numerical keys and string keys: */
     /*     (1) STRING: set(array('a' => 'a_val')) will update current_iteration keys */
     /*     (2) LONG: set(array(0=>array('a'=>'a_val'))) will reset current_iteration_parent */
+    SEPARATE_ARRAY(input_arr);
     input_ht = HASH_OF(input_arr);
     zend_hash_internal_pointer_reset_ex(input_ht, &input_pos);
     first_key_type = zend_hash_get_current_key_ex(input_ht, &key, &index, &input_pos);
